thorlcr/thorutil/thmem.cpp

```diff
@@ -165,6 +165,7 @@ class CSpillable : public CSimpleInterfaceOf<roxiemem::IBufferedRowCallback>
     unsigned spillPriority = SPILL_PRIORITY_DISABLE;
     IThorRowInterfaces *rowIf = nullptr;
     roxiemem::IRowManager *rowManager = nullptr;
+    StringAttr tracingPrefix;
```

Comment:
It would be more efficient if this didn't involve a heap allocation.  I suspect it is a tiny proportion of the time spent in these classes, but I am slightly concerned about the overhead for small numbers of rows in child queries.

Reply:
agree it's a concern, but it's more that if these objects in general are recreated on the heap repeatedly for small sets in child queries, then it is wasteful/inefficient.
I've tried in the past to avoid that, but there is likely plenty of scope to look for further optimizations to minimize reallocation of reusable utility objects like this.
i.e. I think the approach should be, if possible, create a e.g. CSpillableStream object once and reuse it on next iteration of the activity in a child query, in which case the overhead of this StringAttr allocation for example becomes a 1 time overhead and trivial.